src/interactive_creative_file.js

```diff
@@ -1,8 +1,15 @@
-export function createInteractiveCreativeFile() {
+import { parserUtils } from './parser/parser_utils';
+
+export function createInteractiveCreativeFile(
+  interactiveCreativeAttributes = {}
+) {
   return {
-    type: null,
-    apiFramework: null,
-    variableDuration: null,
+    type: interactiveCreativeAttributes.type || null,
+    apiFramework: interactiveCreativeAttributes.apiFramework || null,
+    variableDuration:
+      parserUtils.parseBoolean(
+        interactiveCreativeAttributes.variableDuration
+      ) || null,
```

Comment:
here it will result in `null` if value is false

Reply:
Fixed